### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by SmittieC**
A small refactor:
- Add environments folder
- Add `analytics` and `example` environments
- move `vault.yml` file to `analytics` environment
- Add inventory file to both environments
- Update `production.md` file

### CODE REVIEW

**Comment by SmittieC** [comment]
There other docs seems sort of relavent, but is a bit coonfusing to me tbh. I'm thinking of removing those rather. Thoughts?

File: docs/production.md [hunk 1]
```diff
@@ -1,72 +1,41 @@
...
 
-Add your secrets here. E.g.
-
-```
-# My Project Vault File
-
-vault_default_db_password: SuperSecret
-vault_django_secret_key: als0_SEcr3t
-```
+You can generate a good random key from a command line:
```
Comment (kaapstorm) [inline docs/production.md:37]: Although you didn't write the Python snippet below, there is an easier way to get a good random key:

    $ openssl rand -base64 48

File: docs/production.md [hunk 2]
```diff
@@ -1,72 +1,41 @@
...
 This is where your project-specific configuration will live. 
 
 ```bash
-cp -r inventories/example inventories/myproject
+cp -r environments/example environments/your_environment
 ```
 ### Update Inventory Files
 
-Edit the `hosts.yml` and `vars.yml` files with your project-specific changes.
+Edit the `inventory` and `vault.yml` files with your project-specific details.
```
Comment (kaapstorm) [inline docs/production.md:23]: Could I persuade you to name the inventory file "inventory.ini"? That would be in line with the [filename in commcare-cloud](https://github.com/dimagi/commcare-cloud/blob/master/environments/production/inventory.ini), and the file does use the [INI file format](https://en.wikipedia.org/wiki/INI_file).

```suggestion
Edit the `inventory.ini` and `vault.yml` files with your project-specific details.
```

**Review by kaapstorm — COMMENTED** [review]
These changes are great. I just have one nit, and one suggestion.

File: docs/production.md (hunk 1 above)
Comment (SmittieC) [reply docs/production.md:37]: Great suggestion, thanks!

**Review by kaapstorm — APPROVED** [review]
Thanks!

**Pull request merged**